src/app/modules/bounce/bounce.controller.ts

```diff
@@ -14,9 +14,9 @@ const logger = createLoggerWithLabel(module)
  * @param res - Express response object
  */
 const handleSns = async (
-  req: Request<{}, {}, ISnsNotification>,
+  req: Request<Record<string, string>, string, ISnsNotification>,
   res: Response,
-) => {
+): Promise<Response<never>> => {
```

Comment:
why is this `never`?

Reply:
We don't really care about the return type of `res.send`, and `any` is now linted as a warning. Fine, I guess we can change it to `never` :P

[Further reading](https://blog.logrocket.com/when-to-use-never-and-unknown-in-typescript-5e4d6c5799ad/)

Reply:
great article! thanks for sharing!